client/src/main/webjar/program-details/program-details.html

```diff
@@ -0,0 +1,8 @@
+<md-content flex>
+
+    <p>
+        Program data:
+        <pre ng-bind="programDetails.program | json: 4"></pre>
+    </p>
```

Comment:
Just a dummy placeholder page to prove the route is set up and getting the program correctly.